src/sentry/api/serializers/models/integration_feature.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import IntegrationFeature
+
+
+@register(IntegrationFeature)
+class ServiceHookSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        return {
+            'description': obj.description['data'].strip(),
+            'featureGate': obj.feature_str(),
```

Comment:
I think what you called it in the model is also fine here (`feature`). "feature gate" is sort of the whole process, not this piece of data, imo.

Reply:
I use `featureGate` because that's what the `getsentry` side expects https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/hooks/integrationFeatures.jsx#L26